.syncpackrc

```diff
@@ -33,6 +33,15 @@
 				"**"
 			]
 		},
+		{
+			"dependencies": [
+				"config"
```

Comment:
In syncpack the `filter` attribute means that syncpack will ignore any dependencies that don't match the filter even if they have a version group object like this.

The API is sort of counterintuitive for what we want to achieve right now and there are possibly some alternatives, but for now you'll need to add "config" to the filter regex if you want this to work.

Reply:
Thanks, I missed it.

Addressed in https://github.com/woocommerce/woocommerce/pull/33828/commits/105d95ab6d1b56c8896320d744016db650a242ec

> The API is sort of counterintuitive for what we want to achieve right now

On the other way around: API is intuitive, but our usage is against the design. AFAIU, syncpack by default, tries to keep all packages in sync, then expose `versionGroups` to group the exception rules.

What we do is, ignore everything and sync only a few. This is probably the only way to start cleaning it up gradually, without "do it all" revolution.